WcaOnRails/app/webpacker/components/Delegates/index.jsx

```diff
@@ -35,12 +36,18 @@ export default function Delegates() {
     [delegateGroups],
   );
 
-  const [activeRegion, setActiveRegion] = React.useState();
-  const [adminMode, setAdminMode] = React.useState(false);
+  const [hash, setHash] = useHash();
+
+  const activeRegion = React.useMemo(() => {
+    const selectedRegionIndex = delegateRegions.findIndex((region) => region.id === +hash);
+    if (selectedRegionIndex === -1) {
+      setHash(delegateRegions[0]?.id);
```

Comment:
I was about to approve the PR, and suddenly thought of this edge case. Can you please confirm if http://localhost:3000/delegates#europe is opening the Europe page?

Sorry that I couldn't try it myself because my local DB is bit messed up because of which I can't open anything that uses `roles` table as my local DB has `user_roles` instead of `roles`.

Reply:
I used `region.id` as hash, so http://localhost:3000/delegates#3 is opening Europe page. Should I use region name instead of id?

Reply:
Oops, just realized that there is no `friendly-id` field in metadata. So you can go ahead with ID itself. Will change once the `friendly_id` is added to metadata.